metadata-ingestion/tests/unit/test_transform_dataset.py

```diff
@@ -210,7 +210,8 @@ def test_simple_dataset_ownership_transformation(mock_time):
             "owner_urns": [
                 builder.make_user_urn("person1"),
                 builder.make_user_urn("person2"),
-            ]
+            ],
+            "ownership_type": "DATAOWNER",
```

Comment:
are these defaults necessary - it looks like validate_ownership_type will add this default right?

Reply:
Mainly just want to double check that this change is backwards compatible

Reply:
I removed `"ownership_type"` from this test case, kept it in other one so that we have better coverage of different cases.